ThrowIT/PartyFilterViewController.m

```diff
@@ -0,0 +1,57 @@
+//
+//  PartyFilterViewController.m
+//  ThrowIT
+//
+//  Created by Oore Fasawe on 7/22/22.
+//
+
+#import "PartyFilterViewController.h"
+
+@interface PartyFilterViewController ()
+@property (strong, nonatomic) IBOutlet UILabel *distanceLabel;
+@property (strong, nonatomic) IBOutlet UISlider *distanceSlider;
+@property (strong, nonatomic) IBOutlet UISegmentedControl *attendanceSegmentedController;
+@property (strong, nonatomic) IBOutlet UILabel *attendanceLabel;
+@property (strong, nonatomic) IBOutlet UILabel *ratingLabel;
+@property (strong, nonatomic) IBOutlet UIStepper *ratingStepper;
+
+@end
+
+@implementation PartyFilterViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    [self setDistanceLabeltext];
+    [self setAttendanceLabeltext];
+    [self setRatingLabeltext];
+    // Do any additional setup after loading the view.
+}
+- (IBAction)didPickDistance:(id)sender {
+    [self setDistanceLabeltext];
+}
+
+- (IBAction)didPickAttendance:(id)sender {
+    [self setAttendanceLabeltext];
+}
+- (IBAction)didPickRating:(id)sender {
+    [self setRatingLabeltext];
+}
+
+-(void)setDistanceLabeltext{
+    self.distanceLabel.text = [NSString stringWithFormat:DISTANCELABELTEXTFORMAT, self.distanceSlider.value];
+}
+
+-(void)setAttendanceLabeltext{
+    int attendanceNumbers[] = {0, 1, 2, 3, 4};
```

Comment:
Why do we need this array? Can't we use directly self.attendanceSegmentedController.selectedSegmentIndex?

Reply:
It was mostly for future reasons if I were to check against values like 0, 10, 20, 50, 100 party count limits instead of from just 0-4, but it definitely does not make sense to have them there right now. Will make the needed change. Thank you.